numismatic/feeds/bravenewcoin.py

```diff
@@ -18,15 +22,33 @@ def __init__(self, requester='basic', cache_dir=None, api_key_id=None,
 
         self.headers = {'X-Mashape-Key': f'{self.api_key_secret}'}
 
-    def get_list(self): 
+    def get_digital_currency_symbols(self): 
         api_url = f'{self.api_url}/digital-currency-symbols'
         data = self._make_request(api_url, headers=self.headers)
-        response = []
+        return data['digital_currencies']
+
+    def get_ticker(self, coin, show):
+        api_url = f'{self.api_url}/ticker/'
+        params = dict(coin=coin, show=show)
+        data = self._make_request(api_url, params=params,
+                                    headers=self.headers)
+        return data if data['success'] else None
 
+
+class BraveNewCoinFeed(Feed):
+
+    def __init__(self, requester='basic', cache_dir=None, api_key_id=None,
+                 api_key_secret=None):
+        self.rest_api = BraveNewCoinRestApi(
+            requester=requester, cache_dir=cache_dir, api_key_id=api_key_id,
+            api_key_secret=api_key_secret)
+        self.websocket_api = None
+
+    def get_list(self): 
+        digital_currencies = self.rest_api.get_digital_currency_symbols()
```

Comment:
`digital_currency_symbols` is a public endpoint that BNC exposes. The RestApi just makes this easily accessible from Python. Why should it be private?

Reply:
Okay, I see where you are going, but then it should be implemented in the BNC class not in BNCRestApi class. This extra level of inheritance not only makes it tough to read/follow the code, but it will make it difficult for people to follow EXACTLY what you mean when they add new exchanges